WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -50,7 +50,10 @@
           <ul class="dropdown-menu" role="menu">
             <li><a href="/results/events.php"><%= icon('signal', class: 'fa-fw fa-rotate-90') %> <%= t '.rankings' %></a></li>
             <li><a href="/results/regions.php"><%= icon('trophy', class: 'fa-fw') %> <%= t '.records' %></a></li>
-            <li><a href="<%= persons_path %>"><%= icon('user', class: 'fa-fw') %> <%= t '.persons' %></a></li>
+            <li><a href="<%= persons_path %>"><%= icon('users', class: 'fa-fw') %> <%= t '.persons' %></a></li>
```

Comment:
Ah I wasn't aware that `fa-fw` is Font Awesome native class and we just extend this. I've checked locally and found nice solution anyway. So we can remove all these `fa-fw` classes and change [this](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/assets/stylesheets/navbar-static-top.scss#L8-L10) to:
```scss
  .dropdown-menu .fa {
    @extend .fa-fw;
    margin-right: 10px;
  }
```